### DISCUSSION TITLE
Console.log doesn't work

### DISCUSSION THREAD

**Discussion by Ifra001** [Cyfrin/foundry-full-course-cu#2515] [Q&A]
To fix a test of mine, I ```console.logged``` some of my variables (I think that's what they are called) in my function but they don't show up when i am checking my error in the terminal. I don't think that I am writing them wrong

This is how I am writing them:
```solidity
 console.log("weth value:", wethValue);
```

What could be the issue?

**Comment by EngrPips** [comment]
If your test failed before it got to the point where you used `console.log` then the `console.log` wouldn't be effective at all

**Comment by mumernisar** [comment]
@DontMind-me you probably forgot to add the vvv with it..! more v's give more information about what went wrong.!!

**Comment by EngrPips** [comment]
Interesting

**Comment by Cameadows99** [comment]
my cosole.log didnt work either. I cant remember exactly what I did but I'm almost certain that i just ran "forge clean" in the terminal and it worked

**Reply by EngrPips** [reply]
Wow, Interesting.
